instance/tests/fixtures/api/appserver_detail.json

```diff
@@ -18,7 +18,7 @@
     "edx_platform_commit": "073e9c56a8093691dea782c8b727d839260833b1",
     "openedx_release": "master",
     "use_ephemeral_databases": true,
-    "github_admin_organization_name": "open-craft",
+    "github_admin_organization_name": "[\"open-craft\"]",
```

Comment:
Does the name of this variable need to change based on the new database field name(s)?